### PR TITLE
Create LinkChecker.json

### PR DESCRIPTION
**Pull request by hzik**

### CODE REVIEW

**Comment by ChristopherJennings** [comment]
Can you combine this with with #81?

File: gallery-src/content/elements/LinkChecker.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Link checker",
+  "description": "This custom element gets all links from all rich text elements on your item and checks whether they are live or not.",
+  "thumbnailUrl": "http://amend.cz/link_checker.gif",
```
Comment (ChristopherJennings) [inline gallery-src/content/elements/LinkChecker.json]: Please upload this image to [`/src/data/assets`](https://github.com/Kentico/kontent-custom-element-samples/tree/master/src/data/assets) and update this appropriately. In order to keep the changes in the same PR, be sure to clone this repo locally and change to the `hzik-patch-3` branch.

Comment (ChristopherJennings) [reply gallery-src/content/elements/LinkChecker.json]: Don't forget that it must be named using PascalCase (e.g. LinkChecker.gif)

Comment (hzik) [reply gallery-src/content/elements/LinkChecker.json]: probably done maybe??

Comment (ChristopherJennings) [reply gallery-src/content/elements/LinkChecker.json]: It looks like you uploaded the graphic right in GitHub. This creates a separate PR. You need to clone the repo to your local computer, pull and change to the `hzik-patch-3` branch, add the file locally, then push that branch back up. 

If you need a hand, let me know and we can go through it together.

Comment (hzik) [reply gallery-src/content/elements/LinkChecker.json]: yes please -> the element repo is already at https://github.com/hzik/LinkChecker. Do you also want me to clone the gallery repo so I can upload that gif there??

**Pull request merged**